### PR TITLE
Wrap lines longer than 80 characters in lua_api.txt

### PR DESCRIPTION
**Pull request by rubenwardy**
...and minor formating changes

Based on #1306

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -2398,13 +2425,14 @@ This is basically a reference to a C++ `ServerActiveObject`
...
-    * walk+dig animation key frames
-    * animation frame speed
+* set animation for player model in third person view
+
+        set_local_animation({x=0, y=79}, -- < stand/idle animation key frames
+            {x=168, y=187}, -- < walk animation key frames
+            {x=189, y=198}, -- <  dig animation key frames
+            {x=200, y=219}, -- <  walk+dig animation key frames
+            frame_speed=30): -- <  animation frame speed
+ ^
```
Comment (ShadowNinja) [inline doc/lua_api.txt]: Function name romoved, also `^` added.

Comment (rubenwardy) [inline doc/lua_api.txt]: Wasn't removed, move to code snippet. I'm not sur about this, but this PR is just a rebase of Hoodedice's plus a few line breaks by me.

Comment (ShadowNinja) [inline doc/lua_api.txt]: The function name and params should be on the first line, having an example below doesn't replace that.

Still has a superfluous `^`.

**Comment by paramat** [comment]
Looks good apart from:
Line 2581 needs more indentation.
Line 2428:
I agree with ShadowNinja that the function and params should be on the first line, you could perhaps then have the 'set animation for player model in third person view' line after the example. Or, have the function and params as one line, then 'set animation for player model in third person view', then the example.
The empty lines at 2429 and 2435 are out of character for the file and should be removed, along with that unnecessary '^'.

Please could this pull be updated and merged ASAP as many people need to change this file often.

**Comment by rubenwardy** [comment]
Fixed.

**Comment by paramat** [comment]
Looks fine to me for merge.

**Comment by Zeno-** [comment]
commit 2b189d4507028068694e2d5d2d94ded646d3c604

**Pull request closed without merging**